jetbrains-rider/build.gradle.kts

```diff
@@ -268,12 +283,5 @@ tasks.test {
 tasks.integrationTest {
     useTestNG()
     environment("LOCAL_ENV_RUN", true)
-}
-
-tasks.buildSearchableOptions {
-    enabled = false
-}
-
-tasks.jar {
-    archiveBaseName.set("aws-intellij-toolkit-rider")
+    maxHeapSize = "1024m"
```

Comment:
I see this comes from the above one, should we increase it even more? like 2048 just so we never encounter any future issues?

Reply:
Maybe should increase it for all subplugins...

Reply:
Yeah, I think so. Maybe this is why rider isn't starting correctly? probably not, but it's an easy thing to raise that might cause problems